apps/test/unit/StudioAppTest.js

```diff
@@ -127,5 +127,35 @@ describe("StudioApp", () => {
           item.text === 'footer.how_it_works');
       expect(howItWorksItem.link).to.equal('https://studio.code.org/projects/gamelab/C_2x38fH_jElONWxTLrCHw/edit');
     });
+
+    it("returns How-It-Works item before Report-Abuse item in GameLab", () => {
+      project.getUrl.returns('https://studio.code.org/projects/gamelab/C_2x38fH_jElONWxTLrCHw');
+      var footItems = makeFooterMenuItems();
+      var howItWorksIndex = footItems.findIndex(item => item.text === 'footer.how_it_works');
+      var reportAbuseIndex = footItems.findIndex(item => item.text === 'footer.report_abuse');
+      expect(howItWorksIndex).to.be.below(reportAbuseIndex);
+    });
+
+    it("returns How-It-Works item before Report-Abuse item in AppLab", () => {
+      project.getUrl.returns('https://studio.code.org/projects/applab/l1RTgTXtyo9aUeJF2ZUGmQ');
+      var footItems = makeFooterMenuItems();
```

Comment:
Is this test correct?  I assume this is calling `StudioApp.makeFooterMenuItems` where we probably always call `AppLab.makeFooterMenuItems` in applab, yes?